account_banking_sepa_credit_transfer/tests/test_sct.py

```diff
@@ -353,25 +384,24 @@ def create_invoice(
     ):
         partner_bank = cls.env.ref(partner_bank_xmlid)
         partner_bank.write({"company_id": False})
+        line_data = {
+            "name": "Great service",
+            "account_id": cls.account_expense.id,
+            "price_unit": price_unit,
+            "quantity": 1,
+        }
         data = {
             "partner_id": partner_id,
-            "reference_type": "none",
+            "reference_type": "free",
```

Comment:
Why do you need this change?

Reply:
I changed the value in the module account_payment_order to have the same value ("free") on account.move and account.payment.line for unstructured communication. Avoids a headache for maintainers. Migration scripts are provided in account_payment_order, cf https://github.com/akretion/banking/blob/16-payment_order-improve_and_cleanup/account_payment_order/migrations/16.0.2.0.0/pre-migration.py#L14